internal/juju/models.go

```diff
@@ -150,8 +149,7 @@ func (c *modelsClient) GetModelByName(name string) (*params.ModelInfo, error) {
 
 	modelInfo := results[0].Result
 
-	log.Printf("[DEBUG] Reading model: %s, %+v", name, modelInfo)
-
+	c.Tracef(fmt.Sprintf("Reading model: %s, %+v", name, modelInfo))
```

Comment:
I think "Reading" is misleading verb here, as it means something in the provider context.

```suggestion
	c.Tracef(fmt.Sprintf("Retrieving model info from Juju api: %s, %+v", name, modelInfo))
```